internal/cfgplugins/policyforwarding.go

```diff
@@ -310,8 +317,71 @@ func QosClassificationConfig(t *testing.T, dut *ondatra.DUTDevice) {
 		default:
 			t.Logf("Unsupported vendor %s for native command support for deviation 'qos classification'", dut.Vendor())
 		}
+	} else {
```

Comment:
Why is there qos configuration in the policy forwarding file?  We should not be expanding this function.  Instead please use cfgplugins/qos.go